easy_my_coop_es/views/subscription_request_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="hola">
+        <field name="name">subscription.request.form</field>
+        <field name="model">subscription.request</field>
+        <field name="inherit_id" ref="easy_my_coop.subscription_request_form"/>
+        <field name="arch" type="xml">
+            <field name="lastname" position="after">
+                <field name="vat"/>
+            </field>
+            <field name="subscription_amount" position="after">
+                <field name="voluntary_contribution"/>
+            </field>
```

Comment:
This should probably go in a `easy_my_coop_ voluntary_contribution` module, it's not specific to Spain.

Reply:
I see Houssine is ok to leave it in the "Spain" module for now. So it should be ok.